config/clusters/openscapes/workshop.values.yaml

```
@@ -80,7 +80,7 @@ basehub:
                     cpu_guarantee: 3.725
                     cpu_limit: 3.725
                     node_selector:
-                      node.kubernetes.io/instance-type: r5.xlarge
+                      node.kubernetes.io/instance-type: r5.4xlarge
```

Typo to declare 4x for this profile specifically?

```suggestion
                      node.kubernetes.io/instance-type: r5.16xlarge
```